README.md

```diff
@@ -1 +1,59 @@
-# mstankai
+# mstankai's Toy Repository
+
+I've created this repository to play arround with various projects, and refresh myself on using git. In thefuture I aim to put some small projects and pieces of code here that I have written or will write.
+
+### Things to do for me ...
+
+A to do list:
+
+- [x] Write a README
+- [ ] Practice git:
+  - [ ] Create tasks
+  - [ ] Merging requests
```

Comment:
In github "merge" requests are called "pull requests". Also you can tick that off now :)
```suggestion
  - [x] Create a pull request
```